kivy/config.py

```diff
@@ -874,3 +899,16 @@ def name(self, value):
             Config.write()
         except Exception as e:
             Logger.exception('Core: Error while saving default config file')
+
+    # Load configuration from env
+    if 'KIVY_NO_ENV_CONFIG' not in environ:
+        for key, value in environ.items():
+            if not key.startswith("KCFG_"):
+                continue
+            try:
+                _, section, name = key.split("_", 2)
+            except ValueError:
+                raise
+            section = section.lower()
+            name = name.lower()
+            Config.set(section, name, value)
```

Comment:
Ah, sorry, one more quibble - the lack of a newline here is causing Travis to fail!

(I agree with Travis here.  Every line should end in a carriage return, because then all lines are uniform and your code is easier.)

Thanks again!